tests/test_gaussian.py

```diff
@@ -601,16 +601,14 @@ def circuit(r, phi):
         assert np.allclose(var, expected, atol=tol, rtol=0)
 
         # circuit jacobians
-        gradA = circuit.jacobian([r, phi], method="A")
-        gradF = circuit.jacobian([r, phi], method="F")
+        grad = qml.jacobian(circuit)(r, phi)
```

Comment:
It might be worth creating the qnodes explicitly such that both the analytic and finite diff checks can be kept

Reply:
Yes, the plan is to go back to calculating both analytic and finite diffs (that's why I noticed the difference between `qml.jacobian` and `tape.jacobian`. :slightly_smiling_face: